evm/rlp/sedes.py

```diff
@@ -1,11 +1,52 @@
 from rlp.sedes import (
     BigEndianInt,
     Binary,
+    CountableList,
 )
+from rlp.exceptions import (
+    ListSerializationError,
+    ListDeserializationError,
+)
+
+
+class AccessListElement(CountableList):
+
+    def __init__(self):
+        super().__init__(Binary(max_length=32))
+
+    def serialize(self, obj):
+        result = super().serialize(obj)
```

Comment:
Can we move this to below the `if/else` so that it only happens *if* the validation checks pass.

Reply:
I'd prefer to leave it there, as `super().serialize(obj)` does a lot of checks already. In particular, that `obj` is actually a list and `obj[0]` is binary. If that weren't the case, the error messages would be misleading.

Reply:
That's fine.  I wasn't aware of the  checks that occur in the super call.